cmd/storage-rest-client.go

```diff
@@ -172,6 +172,10 @@ func (client *storageRESTClient) CrawlAndGetDataUsage(ctx context.Context, cache
 	return newCache, newCache.deserialize(b)
 }
 
+func (client *storageRESTClient) GetDiskID() (string, error) {
```

Comment:
you can return error here as no one is supposed to call GetDiskID() on rest client